BikeBase.txt

```diff
@@ -0,0 +1,4 @@
+comanche tomahawk#1300#6#
```

Comment:
Where are the packages?

Reply:
ok.

Reply:
Not ok. Read about packages and naming conventions regarding them. Find examples. Describe to me in the chat (telegram) what you learned before making any changes.